README.md

```diff
@@ -1,85 +1,105 @@
+3. Use terminal B to then run `yarn start`.
+4. A permit URL for both ERC20 and ERC721 is generated in the terminal.
+5. Open the generated permit URL using the link in the console.
+6. Connect your wallet. (This requires that you have imported the two anvil accounts [0] & [1] into your wallet.)
+7. Either the claim or invalidate button should be visible depending on your connected account.
+8. Testing the ERC721 permit is easiest deploying the `nft-rewards` contract from the [repository](https://github.com/ubiquity/nft-rewards)
 
-```
-http://localhost:8080?claim=eyJwZXJtaXQiOnsicGVybWl0dGVkIjp7InRva2VuIjoiMHgxMWZFNEI2QUUxM2QyYTYwNTVDOEQ5Y0Y2NWM1NWJhYzMyQjVkODQ0IiwiYW1vdW50IjoiMTAwMDAwMDAwMDAwMDAwMDAwMCJ9LCJub25jZSI6IjQ0NTUxMjc4NTQwNTU0MzM1MDQ2NzU2NDQ3MzM2MjI1ODg5OTE4OTY5MTczODQwNTU0Nzk2NzQ3MzQzMzAwOTg0NzU4MDIyMzY1ODczIiwiZGVhZGxpbmUiOiIxMTU3OTIwODkyMzczMTYxOTU0MjM1NzA5ODUwMDg2ODc5MDc4NTMyNjk5ODQ2NjU2NDA1NjQwMzk0NTc1ODQwMDc5MTMxMjk2Mzk5MzUifSwidHJhbnNmZXJEZXRhaWxzIjp7InRvIjoiMHhjODZhMDU5NzgwMThlMDRkNmVGMmFhNzNFNjlhNzMzQzA2ZDFmODllIiwicmVxdWVzdGVkQW1vdW50IjoiMTAwMDAwMDAwMDAwMDAwMDAwMCJ9LCJvd25lciI6IjB4NTRmNGEzNjQyMkRjOTZkMDg0OTY3NWMxZjBkZDJCOTZEMjc1NThFMiIsInNpZ25hdHVyZSI6IjB4NWI0OTE5MjhmYzI4MzBlMjZiNTViMWUxOWQ3YzVhMmVjNGE2ZmRhYWI1OGFiYjgyOWMwNmYzYzlkNGE4YTc5YjAzYmE2NjlkMDM4YjFmYzg5NjgzYzMyYjBiYTA5MzU2MDRjMGU1MDNjYWE3ZmY2ZWM2MDg2ZWZlYjY2MTY5MjQxYyJ9
-```
+- Importing the anvil accounts into your wallet:
+  - Open your wallet provider and select `import wallet` or `import account`, something to that effect.
+  - You can obtain the private keys by simply running the yarn command or just `anvil` and it will list the private keys for you.
+  - Copy and paste into your wallet provider and you should now be able to interact with the locally generated permits.
+  - Only account [0] can claim it and only account [1] can invalidate it.
 
-3. Admin posts offline permit URL in issue comments
-4. Bounty hunter opens permit URL, connects wallet and clicks a "withdraw" button to get a payment
+## What to expect
 
-## How to test locally
+#### Setup
 
-1. Set `.env` variables.
-2. Run `anvil --chain-id 31337 --fork-url https://rpc.gnosis.gateway.fm` in a separate terminal.
-3. Run the Anvil commands (uses the Anvil default wallets).
-4. In the project root run `yarn start`.
-5. A permit URL for both ERC20 and ERC721 is generated in the terminal.
-6. Open the generated permit URL defaulting to the variable values in the `.env` file.
-7. Connect the bounty hunter's address.
-8. Click the "withdraw" button to get a reward.
-9. Testing the ERC721 permit is easiest deploying the `nft-rewards` contract from the [repository](https://github.com/ubiquity/nft-rewards)
+- Using the yarn command setup it's expected that you will have a local blockchain instance spun-up for you. It will be a local shallow copy of the Gnosis blockchain which will allow you to test the permit generation, claiming and invalidation features.
```

Comment:
```suggestion
- Using the yarn command setup it's expected that you will have a local blockchain instance spun-up for you. It will be a local shallow copy of Gnosis Chain which will allow you to test the permit generation, claiming and invalidation features.
```